packages/atomic/src/components/facets-v1/facet-value-box/facet-value-box.tsx

```diff
@@ -1,10 +1,40 @@
 import {FunctionalComponent, h} from '@stencil/core';
+import {FacetValueProps} from '../facet-common';
 
-interface FacetValueBoxProps {
-  value: string;
-}
+export const FacetValueBox: FunctionalComponent<FacetValueProps> = (props) => {
+  const count = props.numberOfResults.toLocaleString(props.i18n.language);
+  const ariaLabel = props.i18n.t('facetValue', {
+    value: props.displayValue,
+    count: props.numberOfResults,
+  });
 
-// TODO: add interactivity, style & part
-export const FacetValueBox: FunctionalComponent<FacetValueBoxProps> = (
-  props
-) => <div class="border border-primary">{props.value}</div>;
+  return (
+    <li key={props.displayValue}>
+      <button
```

Comment:
I'm wondering if `button` is the correct element from an accessibility perspective?

I feel the element should still be a input configured as a checkbox, even if visually it looks like a box.
Similarly, when in single-select mode, I believe the html should use a radio list.

Reply:
I believe the button is the correct element. Here I haven't set the role to checkbox, but I could, now it's currently seen as a toggle button by the screen reader, indicated if it's toggled or not. Since it's not visually a checkbox, I wouldn't want to confuse users.

For the single select, I have not used the radio group because I don't want the facet to be used differently than others, the radio list makes it quick to switch between selection using the arrow keys, here it might only trigger a bunch of queries, since it's not really a typical form. From a UX perpective, they're more navigation links

Reply:
Alright, for what it's worth I saw [Amazon](https://www.amazon.ca/s?k=laptops&i=electronics&rh=n%3A667823011&dc&language=fr&__mk_fr_CA=%C3%85M%C3%85%C5%BD%C3%95%C3%91&qid=1623278841&ref=sr_ex_p_36_0) and [Google](https://www.google.com/search?tbm=shop&q=laptop&tbs=vw:l,mr:1,price:1,ppr_max:300&sa=X&ved=0ahUKEwi3rfLu0ovxAhXdB50JHXXmAtoQvSsI9AMoAA&biw=923&bih=754) are both using `li` tags with `span` tags instead of `button` tags. Not sure if this is a coincidence, but either way, I would not have expected them to implement them this way,